docs/api/hardhat/hardhat-zksync-upgradable.md

```diff
@@ -6,7 +6,6 @@ The plugin is based on [@openzeppelin/hardhat-upgrades](https://www.npmjs.com/pa
 
 ::: info Overview
 - The hardhat-zksync-upgradable plugin supports transparent upgradable proxies and beacon proxies.
```

Comment:
```suggestion
- The hardhat-zksync-upgradable plugin supports transparent upgradable proxies and beacon proxies.
- UUPS proxies are not supported yet.
- Proxy upgrade validations are not supported yet.
```

Reply:
@FilipTxFusion Just noticed we didn't have the info about validations here. If this is accurate, please merge as well.